src/Tools/Common/Commands/Utils.cs

```diff
@@ -27,15 +34,15 @@ public static int FindProcessIdWithName(string name)
                 {
                     if (commonId != -1)
                     {
-                        Console.WriteLine("There are more than one active processes with the given name: {0}", name);
+                        Console.WriteLine($"There are more than one active processes with the given name: {name}");
```

Comment:
Rather than embedding console output within helper calls how about:

```suggestion
                        throw new CommandLineErrorException($"There are more than one active processes with the given name: {name}");
```

Then callers need to catch the exception and write to the console. That allows this type to remain static with no console injected.

Reply:
Thanks! Changed all error writes except for LaunchDSRouterProcess to throw a CommandLineErrorException, and updated callers to try/catch.